docs/containerd-env-setup.md

```diff
@@ -52,8 +52,10 @@ Then start `containerd-nydus-grpc` remote snapshotter:
     --config-path /etc/nydusd-config.json \
     --log-level debug \
     --root /var/lib/containerd/io.containerd.snapshotter.v1.nydus \
+    --cache-dir /var/lib/nydus/cache \
     --address /run/containerd/containerd-nydus-grpc.sock
 ```
+`cache-dir` argument represent the blob cache root dir, if not set, it will be set `root` + "/cache". And `work_dir` in nydusd-config.json will be no use.
```

Comment:
How about rephrasing it like:
`cache-dir` argument represents the blob cache root dir, if unset, it will be set `root` + "/cache". It overrides the `work_dir` option in nydusd-config.json.